lua/asllib.lua

```diff
@@ -4,6 +4,16 @@
 --
 Asllib = {} -- how should we refer to the below?
 
+-- clamp to bounds, must be greater than zero
+function boundgz( n )
+    if type(n) == 'function' then
+        return function()
+            nn = n()
+            return (nn <= 0.01) and 0.01 or nn
```

Comment:
curious how you settled on `0.01` to clamp to?  might be useful to doc this somewhere?